doc/whats-new.md

```diff
@@ -1,356 +1,197 @@
+integration distances*. This is expected to have a substantial impact on the
+calculation time, since small magnitude ruptures will affects a lot
+less sites than before. In order to use the feature the user has to
+specify in the `job.ini` file a list of pairs (magnitude, distance) for each
+tectonic region type. In the past the user would write
+
+`maximum_distance = {'Active Shallow Crust': 200}`
+
+and the engine would use an integration distance of 200 km for all
+ruptures of the given tectonic region type, disregarding the magnitude.
+Nowadays, the user can write
+
+`maximum_distance = {'Active Shallow Crust': [(8, 200), (7, 100), (5, 20)]}`
+
+and the engine will use an integration distance of 200 km for ruptures
+of magnitude 8, of 100 km for ruptures of magnitude 7, and of 20 km
+for ruptures of magnitude 5. Intermediate magnitudes will use a
+linearly interpolated distance. It is up to the user to provide a
+sound magnitude-distance function. For the future, we plan to release
+tools to help the users in the task of inferring magnitude-distance functions.
+If in doubt, you can always use the old syntax, equivalent to a
+constant maximum distance for all magnitudes, and the engine will keep
+working as in the past.
+
+A lot of work went into the UCERF calculators, but they are still officially
+marked as experimental and are left undocumented on purpose. There is a
+new time-dependent classical calculator, while the old calculators were
+substantially improved.
+
+There is an experimental command `run_tiles` to split a `sites.csv`
+file in tiles and run multiple calculations at once. This for expert
+use only.
+
+There is a new command `oq reset` that will remove all calculations of
+the current user from the database and the filesystem. It is meant for
+cleaning up a testing machine were there are no important calculations.
+Use it with care.
+
+New features in hazardlib
+--------------------------
+
+New GMPEs (site adjusted Pezeshk et al. 2011) were added.
+
+The new `GriddedSurface` object introduced in engine 2.2 can now be
+serialized in the NRML 0.5 format properly.
+
+The calculator `calc_hazard_curves` now supports mutually exclusive
+sources and ruptures too.
+
+Performance improvements
+------------------------
+
+The calculation of ground motion fields is a lot faster than before in
+situations with a large amount of hazard sites. The reason is that now
+the distances are computed in the workers, whereas before they were
+computed in the controller node. This has a significant impact on
+users with a cluster.
+
+Changes
+------------
+
+There are two new limits in the event based calculators:
+
+1. it is impossible to run calculations with more than 65536 tasks
+2. it is impossible to have tasks generating more than 65536 events each
+
+The limits are there for safety purpose: users trying to run calculations
+impossibly large will get an error early, rather than trashing their machines.
```

Comment:
These limits have been added to prevent users from running impossibly large calculations: an error message will be displayed early rather than causing machines to crash.